site/docs/types/1/0110-Actors.md

```diff
@@ -0,0 +1,48 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project. -->
+
+# 0110 Actors
+
+Most metadata repositories are run in a secure mode
+requiring incoming requests to include the requester’s
+security credentials.
+Therefore we have an identifier for each unique logged on
+security identity (aka userId).
+This identity is recorded with specific entities and
+relationships when they are created or updated.
+The userId for a server is also captured in the metadata
+model (see [0040 Servers in Area 0](0040-Software-Servers.md))
+so it is possible to
+correlate the actions of a data processing server with
+changes to the metadata.
+
+Figure 1 captures how people and teams are represented.
+
+![UML](0110-Actors.svg#pagewidth)
+> Figure 1: Collecting information about people and teams
+
+**UserIdentity** provides a structure for storing the security
```

Comment:
Create a header for the type name, and then italicise (not bold) the first mention of the type below that heading. So this example would be:

```suggestion
## UserIdentity

*UserIdentity* provides a structure for storing the security...
```

This will create an anchor link in the doc for UserIdentity so that we can directly link to the description from any other page. (Of course the same applies to each other type in this page, too.)

Reply:
Done. Updated as per your request.